<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingwidget</class>
 <widget class="QWidget" name="settingwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Easy">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Medium">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Hard">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Jewel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Fruit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Vegetable">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hell">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Volume">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>491</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/slider-track-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_settings">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Broadway&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/dialogtitle.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>381</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/DialogButtonGlow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>289</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Level&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Settingpage/DialogButtonGlow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Hell</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Jewel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Fruit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Vegetable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/blueBar.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_volume_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_volume_2</zorder>
  <zorder>label</zorder>
  <zorder>Easy</zorder>
  <zorder>Medium</zorder>
  <zorder>Hard</zorder>
  <zorder>Jewel</zorder>
  <zorder>Fruit</zorder>
  <zorder>Vegetable</zorder>
  <zorder>hell</zorder>
  <zorder>Volume</zorder>
  <zorder>label_3</zorder>
  <zorder>label_volume</zorder>
  <zorder>label_settings</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
